lib/inputstreamhelper/utils.py

```diff
@@ -261,14 +261,15 @@ def arch():
 
 def hardlink(src, dest):
     """Hardlink a file when possible, copy when needed"""
-    from os import link
+    
 
```

Comment:
Please remove these 2 lines completely, the rest seems fine.

Reply:
Fixed it.